app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt

```diff
@@ -1911,6 +1911,309 @@ class DatabaseMigrationAndroidTest {
+
+    val phoneNumberTableName = "PatientPhoneNumber"
+    val patientTable = "Patient"
+    val patientUuid = "patientUuid"
+    val phoneUuid = "phoneUuid"
+
+    db_v35.execSQL("""
+      INSERT INTO "PatientAddress" VALUES(
+        'addressUuid',
+        'colony or village',
+        'district',
+        'state',
+        'country',
+        'created-at',
+        'updated-at',
+        NULL,
+        'recorded-at')
+    """)
+
+    db_v35.execSQL("""
+      INSERT INTO $patientTable VALUES(
+        '$patientUuid',
+        'addressUuid',
+        'patient-name',
+        'name',
+        'MALE',
+        NULL,
+        'ACTIVE',
+        'created-at',
+        'updated-at',
+        'null',
+        'IN_FLIGHT',
+        'recorded-at',
+        25,
+        'age-updated-at',
+        'dob');
+    """)
+
+    db_v35.execSQL("""
+      INSERT INTO $phoneNumberTableName VALUES(
+      '$phoneUuid',
+      '$patientUuid',
+      '981615191',
+      'mobile',
+      0,
+      'created-at',
+      'updated-at',
+      'null',
+      'recorded-at')
+    """)
+
+    val db_v36 = helper.migrateTo(version = 36)
+
+    db_v36.query("SELECT * FROM $patientTable").use {
+      it.moveToNext()
+      assertThat(it.count).isEqualTo(1)
+      assertThat(it.string("uuid")).isEqualTo(patientUuid)
+    }
+
+    db_v36.query("SELECT * FROM $phoneNumberTableName").use {
+      it.moveToNext()
+      assertThat(it.count).isEqualTo(1)
+      assertThat(it.columnCount).isEqualTo(8)
+      assertThat(it.string("patientUuid")).isEqualTo(patientUuid)
+      assertThat(it.string("uuid")).isEqualTo(phoneUuid)
+    }
+  }
+
+  @Test
+  fun migrate_business_ids_from_35_to_36(){
+    val db_v35 = helper.createDatabase(version = 35)
+
+    val patientTable = "Patient"
+    val businessIdTable = "BusinessId"
+    val patientUuid = "patientUuid"
+    val businessUuid = "businessUuid"
+
+    db_v35.execSQL("""
+      INSERT INTO "PatientAddress" VALUES(
+        'addressUuid',
+        'colony or village',
+        'district',
+        'state',
+        'country',
+        'created-at',
+        'updated-at',
+        NULL,
+        'recorded-at')
+    """)
+
+    db_v35.execSQL("""
+      INSERT INTO $patientTable VALUES(
+        '$patientUuid',
+        'addressUuid',
+        'patient-name',
+        'name',
+        'MALE',
+        NULL,
+        'ACTIVE',
+        'created-at',
+        'updated-at',
+        'null',
+        'IN_FLIGHT',
+        'recorded-at',
+        25,
+        'age-updated-at',
+        'dob')
+    """)
+
+    db_v35.execSQL("""
+      INSERT INTO $businessIdTable VALUES (
+      '$businessUuid',
+      'patientUuid',
+      'meta-version',
+      'data',
+      'created-at',
+      'updated-at',
+      'null',
+      'recorded-at',
+      'simple-uuid',
+      'bp-passport'
+      )
+    """)
+
+    val db_v36 = helper.migrateTo(version = 36)
+
+    db_v36.query("SELECT * FROM $patientTable").use {
```

Comment:
Is this verification necessary? There is no foreign key from patient to business ID.

Reply:
I will remove this.